packages/babylon/src/parser/expression.js

```diff
@@ -968,7 +968,10 @@ pp.parseAwait = function (node) {
   if (this.isLineTerminator()) {
     this.unexpected();
   }
-  node.all = this.eat(tt.star);
+  node.all = false;
```

Comment:
Can remove this and then fixup the tests (since otherwise it's node.all being undefined)